ckanext/dgu/lib/reports.py

```diff
@@ -569,7 +574,8 @@ def admin_editor(org=None, include_sub_organizations=False):
 
         for g in q.all():
             record = {}
-            record['publisher'] = g.name
+            record['publisher_id'] = g.name
```

Comment:
Can you change _id to _name, to be consistent with our model please? publisher.id is something else again.